### DISCUSSION TITLE
Welcome to msfs2020-google-map Discussions!

### DISCUSSION THREAD

**Discussion by derekhe** [derekhe/msfs2020-google-map#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dedebis** [comment]
Bonjour...j'ai l'impression que les tuiles téléchargées par le programme ne vont pas au bon répertoire !
Quel est le chemin de téléchargement ? j'ai toujours les bâtiments bâtiments !!!
exemple : Fourvière a Lyon , Franc
![6](https://user-images.githubusercontent.com/94559107/144093473-f0910b1c-69ed-4191-8fc1-2308a45028f8.jpg)
e

**Comment by Dedebis** [comment]
re bonsoir...
je n'ai pas l'habitude de ce site...enfin si quelqu'un pouvait me répondre !

![10](https://user-images.githubusercontent.com/94559107/144099083-dca3fa19-dd16-46b1-9c0f-94181913756f.jpg)

**Reply by derekhe** [reply]
Sorry, could you describe your problem better?

**Comment by Dedebis** [comment]
Bonjour
Votre programme a l'air de télécharger des tuiles mais c'est toujours le
base Bing qui s'affiche à l'écran !
Ma question: quel répertoire utiliser vous ?
Comme je vous l'ai déjà dit, MSFS est installé sur le disque C: et les
scènes sur le disque D:
Dans la fenêtre cmd je ne vois pas ou sont envoyer les fichiers jpeg que
vous téléchargez ?
Je ne sais plus quoi vous dire, les copies d'écran et les fichiers que je
vous ai envoyés sont explicite...

**Comment by Dedebis** [comment]
[image: 4.jpg]
Ma config MSFS des données

[image: 5.jpg]

msfs2020-google-map-2021-12-01 lancé...

Peut-on avoir une information sur le cache ?

[image: 6.jpg]



Le ven. 3 déc. 2021 à 07:11, dede dedebis ***@***.***> a écrit :

> Bonjour
> Votre programme a l'air de télécharger des tuiles mais c'est toujours le
> base Bing qui s'affiche à l'écran !
> Ma question: quel répertoire utiliser vous ?
> Comme je vous l'ai déjà dit, MSFS est installé sur le disque C: et les
> scènes sur le disque D:
> Dans la fenêtre cmd je ne vois pas ou sont envoyer les fichiers jpeg que
> vous téléchargez ?
> Je ne sais plus quoi vous dire, les copies d'écran et les fichiers que je
> vous ai envoyés sont explicite...
>
>
>
> Le ven. 3 déc. 2021 à 06:04, hesicong ***@***.***> a écrit :
>
>> Sorry, could you describe your problem better?
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/derekhe/msfs2020-google-map/discussions/4#discussioncomment-1741814>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AWRNXA6UFGZAZOMR3SOV4HLUPBFWZANCNFSM5CMECSMA>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>>
>


D:\msfs2020-google-map-2021-12-01>.\Python39\python.exe src\app.py
Restoring hosts
Restored hosts
Restoring hosts
Restored hosts
root "Autorités de certification racines de confiance"
CertUtil: -delstore La commande s’est terminée correctement.
root "Autorités de certification racines de confiance"
Certificats connexes :

Concordance exacte :
Élément 5 :
Numéro de série : c9dfffaa7d7e8ef1c6bc38e6781f0fb1
Émetteur: CN=mkcert ***@***.***, ***@***.***, O=mkcert development CA
 NotBefore : 18/08/2021 15:01
 NotAfter : 18/11/2023 15:01
Objet: ***@***.***, O=mkcert development certificate
Il ne s’agit pas d’un certificat racine
Hach. cert. (sha1) : ae4cfdbea30eaa635443b6914b3b863b52f693de

Le certificat ***@***.***" était déjà dans le magasin.
CertUtil: -addstore La commande s’est terminée correctement.
Overriding hosts
Hosts override
 * Serving Flask app 'server' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://192.168.1.31:39871/ (Press CTRL+C to quit)
Checking mock server health
127.0.0.1 - - [03/Dec/2021 07:50:00] "GET /health HTTP/1.0" 200 -
Mock server health passed
Checking nginx server health
Health check passed

**Comment by derekhe** [comment]
Hi, please try post the screenshots in https://github.com/derekhe/msfs2020-google-map/discussions/4, don't directly reply in email otherwise I can't see your images.

**Comment by Dedebis** [comment]
Je ne peux pas mettre de fichiers dans "#4" pouvez vous me repondre dans ma
boite mail : ***@***.*** en dehors du site https://github.com/...

je ne comprends rien à son fonctionnement ! merci @+
